### ISSUE TITLE
Team description editing problem

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#295]
**Reported by Krunchin Keith  USA  on 28 Oct 37493445 19:06 UTC**

This problem is happening on a lot of projects lately.

It is not just me, others team founders are reporting same. Project admins are baffled.

On projects that used to display the team page as proper html, it now displays as <BR> (instead of a break) and in view source it looks like &lt;BR&gt;

here's a test I did across 4 projects:

At APS@home my team page was ok, I went to management functions and edited it.

I manually type in TEST <BR> TEST <BR> TEST <BR> in front of what was already there.

Then saved.

Then edited again and did view source.

It shows

TEST &lt;BR&gt; TEST &lt;BR&gt; TEST &lt;BR&gt; ...... and all the other code has been changed too.

Same thing on a different project BRaTS@home, but I used a different browser.

And same thing on a different project, PS3GRID using this same browser.

They all displayed correctly before.

If I view source of team management (team_edit_form.php) on a project I haven't edited recently it shows as <BR> in the source and not as &lt;BR&gt;.

I went to an old project that does not have updated code, tried same edits and it remained as it should be, proper html.

Someone has introduced something that causes this. As all the ones fubar'd have a more recent date, May 9th or 15th tag in them.

New projects, with the problem say (or similar):
```
<!-- SVN VERSIONS -->
<!-- $Id: util.inc 12691 2007-05-18 15:02:02Z Rytis $ -->
<!-- $Id: translation.inc 12673 2007-05-15 21:41:59Z boincadm $ -->
```
Old one without the problem says:
```
<!-- CVS VERSIONS -->
<!-- $Id: util.inc,v 1.101 2006/02/20 19:11:48 ballen Exp $ -->
<!-- $Id: translation.inc,v 1.18 2005/10/29 05:30:55 boincadm Exp $ -->
```
So somewhere, probably more recently there was introduced an error.

**Comment by romw** [comment]
**Commented by Rytis on 30 Oct 37493978 17:20 UTC**
(In [Fix #303 (was introduced in [12691](13052])))

**Comment by romw** [comment]
**Commented by Nicolas on 24 May 37494026 09:46 UTC**
Formatting fixed on description (if you have a line with a space alone on it, it doesn't separate the paragraph; the line has to be totally empty).
